internal/charmstore/common_test.go

```diff
@@ -4,11 +4,13 @@
 package charmstore // import "gopkg.in/juju/charmstore.v5-unstable/internal/charmstore"
```

Comment:
Saw this elsewhere & maybe I don't understand why it's there? Is this an erroneous comment?